ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -239,6 +240,31 @@ const moduleLabels: Map<AccessModule, JSX.Element> = new Map([
   [AccessModule.DATAUSERCODEOFCONDUCT, <div>Sign Data User Code of Conduct</div>],
 ]);
 
+const withAborter = async(abortableFn: (AbortController) => void) => {
```

Comment:
Hmm, what is the use case for this? In the usages below, I don't follow in what scenarios using an AbortController like this would matter. If the API call fails, aborting the request is probably unnecessary AFAICT. We're also not using the abort call later, e.g. for a `useEffect` / dismount cleanup.

Reply:
good point - a few iterations back, this was used in a useEffect cleanup.  It's not anymore, so this is kinda pointless.

Reply:
removed.